cwltool/process.py

```diff
@@ -44,7 +45,8 @@
                                 "ShellCommandRequirement",
                                 "StepInputExpressionRequirement",
                                 "ResourceRequirement",
-                                "InitialWorkDirRequirement"]
+                                "InitialWorkDirRequirement",
+                                "LoadListingRequirement"]
```

Comment:
Can we have a test for using `LoadListingRequirement` without passing the `--enable-ext` flag? Should behave the same way `cwltool` does w/o this PR